extensions/wikia/CreateNewWiki/css/_variables.scss

```diff
@@ -2,6 +2,7 @@ $content-spacing-xxsmall: 6px;
 $content-spacing-xsmall: 10px;
 $content-spacing-small: 12px;
 $content-spacing-small-plus: 14px;
+$content-spacing-regular-minus: 16px;
```

Comment:
This name sounds weird for me ;) I see that `$content-spacing-xsmall` and `$content-spacing-xlarge` are never used so maybe we can use that names?